openshift/ci-operator/build-image/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Dockerfile to bootstrap build and test in openshift-ci
 
-FROM docker.io/openshift/origin-release:golang-1.14
+FROM docker.io/openshift/origin-release:golang-1.15
```

Comment:
I think we need to changes these lines too:
https://github.com/openshift-knative/serverless-operator/blob/release-1.14/knative-operator/Dockerfile#L1
https://github.com/openshift-knative/serverless-operator/blob/release-1.14/openshift-knative-operator/Dockerfile#L1
Otherwise this looks good to me.

Reply:
These files are not part of the templating mechanism. Maybe good to add it there.

Reply:
Actually, it's probably the right time to change the base image to the ones from OpenShift CI: registry.ci.openshift.org/openshift
Similar to https://github.com/openshift-knative/serverless-operator/commit/1176cb1fe5d6442f431dea93ed5da42a40f99deb and https://github.com/openshift-knative/serverless-operator/commit/aa7513338efe92c81d2060b7aa52d718cb6d391f